go/store/datas/tag_test.go

```diff
@@ -43,6 +43,7 @@ func TestNewTag(t *testing.T) {
 	cmRef, err := types.NewRef(commit, types.Format_7_18)
 	assert.NoError(err)
 	tag, err := NewTag(context.Background(), cmRef, types.EmptyStruct(types.Format_7_18))
+	assert.NoError(err)
```

Comment:
Needs to be require

Reply:
Changed it. Want me to swap all over `tag_test.go` over to `require` instead of assert?